crates/zeta/src/init.rs

```diff
@@ -21,6 +22,8 @@ actions!(
 );
 
 pub fn init(cx: &mut App) {
+    feature_gate_predict_edits_rating_actions(cx);
```

Comment:
Could we rename this now that it's no longer specifically about the actions for rating suggestions?

```suggestion
    feature_gate_predict_edits_actions(cx);
```